pyknit/Chart.py

```diff
@@ -13,11 +13,61 @@
 from PIL import Image, ImageDraw, ImageFont
 from typing import Set
 
+stitch_legend = {  # Default legend. Incomplete for now.
+    "k": {
+        "instruction": "knit",
+        "symbol": " ",
+        "width": 1,  
+    },
+    "kfb": {
+        "instruction": "knit front and back",
+        "symbol": "V",
+        "width": 1,
+    },
+    "k2tog": {
+        "instruction": "knit two together",
+        "symbol": "/",
+        "width": 1,
+    },
+    "r": {
+        "instruction": "r",  # instruction?
```

Comment:
There actually is no standard r instruction, I just threw that in for testing.  I don't mind keeping it for testing, but it probably doesn't need to be in a standard instruction set.  

If you want to replace it rather than remove it:

```
    "yo": {
        "instruction": "yarn over",
        "symbol": "O",
        "width": 1
    },
```

would be a valid and useful replacement.

Reply:
Added.